builder/builder.go

```diff
@@ -81,3 +111,48 @@ func BuildImage() error {
 
 	return err
 }
+
+func PushImageToHub(authData dockerAuthData) error {
+
+	// TODO: setup ssh keys for logging into docker hub
+
+	buildContext := context.Background()
+	cli, err := client.NewEnvClient()
```

Comment:
`cli` is misleading as it also means command-line interface. Maybe have it as `dockerClient`?

Reply:
Done